src/e2e-test/features/spanner/sink/GCSToSpanner.feature

```diff
@@ -30,14 +34,18 @@ Feature: Spanner Sink - Verification of GCS to Spanner data transfer
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
 
-  @GCS_CSV_TEST @SPANNER_SINK_NEWDB_TEST
-  Scenario: To verify data is getting transferred successfully from GCS to Spanner new database
```

Comment:
Do not change existing scenario for @SPANNER_SINK_NEWDB_TEST. Keep this scenario as it is.
Scenario: To verify data is getting transferred successfully from GCS to Spanner new database